setup.py

```diff
@@ -1,6 +1,38 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='nowcasting_dataset',
-    version='0.1',
-    packages=find_packages())
+    name="nowcasting_dataset",
+    version="0.1.1",
+    license="MIT",
+    description="Nowcasting Dataset",
+    author="Jack Kelly, Peter Dudfield, Jacob Bieker",
+    author_email="info@openclimatefix.org",
+    company="Open Climate Fix Ltd",
+    install_requires=[
+        "numpy",
+        "pandas",
+        "matplotlib",
+        "zarr",
+        "xarray",
+        "ipykernel",
+        "h5netcdf",
+        "gcsfs",
+        "scikit-image",
+        "torch",
+        "pytorch-lightning",
+        "dask",
+        "pvlib",
+        "pyproj",
+        "flake8",
+        "jedi",
+        "mypy",
+        "tables",
+        "boto3",
+        "moto",
+        "neptune-client",
+        "pydantic",
+        "pytest-cov",
+        "plotly",
+    ],
+    packages=find_packages(),
+)
```

Comment:
Should we add a long description here? So it comes up on pypi

Reply:
Just added it!